ANMN/LTSP/TSaggregator/TSagg_metadata.json

```diff
@@ -362,6 +321,6 @@
         "references": "http://www.imos.org.au",
         "source": "Mooring",
         "standard_name_vocabulary": "NetCDF Climate and Forecast (CF) Metadata Convention Standard Name Table 45",
-        "lineage": "The variable of interest (VoI) is produced by sequentially concatenating the individual values in each of the input files. The resulting variable has dimension OBS. The VoI's ancillary_variables, in particular the corresponding quality-control flags, are also included, with dimension OBS. If the quality control variable is absent in any input file, the corresponding flags in the output file will be set to 0 ('no QC performed'). The variable TIME from input files is concatenated into a variable TIME(OBS). This could result in a non-uniform time interval. The DEPTH variable from input files is concatenated into a variable DEPTH(OBS). If not present, fill values are stored. DEPTH_quality_control, if present, is also included. Where  the DEPTH variable is absent, the corresponding DEPTH_quality_control values are set to 9 ('missing value'). All output variables with the OBS dimension are sorted in chronological order. In order to keep track of the provenance of VoI in the aggregated file, accessory variables are created. Product created with https://github.com/aodn/..."
+        "lineage": "The variable of interest (VoI) is produced by sequentially concatenating the individual values in each of the input files. The resulting variable has dimension OBS. The VoI's ancillary_variables, in particular the corresponding quality-control flags, are also included, with dimension OBS. If the quality control variable is absent in any input file, the corresponding flags in the output file will be set to 0 ('no QC performed'). The variable TIME from input files is concatenated into a variable TIME(OBS). This could result in a non-uniform time interval. The DEPTH variable from input files is concatenated into a variable DEPTH(OBS). If not present, fill values are stored. DEPTH_quality_control, if present, is also included. Where  the DEPTH variable is absent, the corresponding DEPTH_quality_control values are set to 9 ('missing value'). All output variables with the OBS dimension are sorted in chronological order. In order to keep track of the provenance of VoI in the aggregated file, accessory variables are created."
```

Comment:
Sorry, but I think you still have the old version of the text here. It still refers to the OBS dimension, and doesn't mention PRES & PRES_REL.